manifests/params.pp

```diff
@@ -25,7 +25,13 @@
     }
     'RedHat': {
       $package          = 'tftp-server'
-      $daemon           = false
+      if $::operatingsystemrelease =~ /^7/ {
```

Comment:
I think you're right about EL6. Fedora 23 and 24 both have the `tftp.socket`.
Should I change it?